torch/testing/_internal/distributed/rpc/rpc_test.py

```diff
@@ -4794,6 +4798,10 @@ def _test_rref_proxy_timeout(self, rref_proxy_api):
         with self.assertRaisesRegex(RuntimeError, expected_error):
             rref_api(timeout=timeout).my_instance_method(torch.ones(2, 2))
 
+        # FIXME We wait until the remote completed creating the OwnerRRef
+        # because there's currently a race if we shut down RPC before that.
+        slow_rref.to_here()
```

Comment:
cc @rohan-varma 

Is the `_get_future` API sufficient for this purpose? What's the contract on that API? If user A share it with user B, will this API work on user B?

https://github.com/pytorch/pytorch/blob/4aa91871c0364279b2f3d04c1ef5594ef1e17f58/torch/csrc/distributed/rpc/init.cpp#L442-L452

Reply:
I've used `slow_rref.to_here()` because it's being used for this same exact purpose a few lines above:
https://github.com/pytorch/pytorch/blob/4d7abdbdadd440cb4b8412f1e309cae14a687b49/torch/testing/_internal/distributed/rpc/rpc_test.py#L4772-L4773